### PR TITLE
[doc] [tor] Clarify when to use bind

### PR DESCRIPTION
**Pull request by maflcko**
Not sure if useful but this is a summary of #6585.

### CODE REVIEW

**Comment by sipa** [comment]
ACK

**Comment by fanquake** [comment]
ACK

**Comment by laanwj** [comment]
Makes sense, thanks

File: doc/tor.md [hunk 1]
```diff
@@ -70,9 +70,14 @@ In a typical situation, where you're only reachable via Tor, this should suffice
...
 	./bitcoind -proxy=127.0.0.1:9050 -externalip=57qr3yd1nyntf5k.onion -listen
 
-(obviously, replace the Onion address with your own). If you don't care too much
-about hiding your node, and want to be reachable on IPv4 as well, additionally
-specify:
+(obviously, replace the Onion address with your own). It should be noted that you still
+listen on all devices and another node could establish a clearnet connection, when knowing
+your address. To mitigate this, additionally bind the address of your Tor proxy:
+
+	./bitcoind ... -bind=127.0.0.1
```
Comment (Diapolo) [inline doc/tor.md]: AFAIK using -bind will implicitly set -listen, too. Dunno if that requires further clarification.

**Pull request merged**